test/test_support.ts

```diff
@@ -159,113 +172,64 @@ export function getLineAndColumn(source: string, token: string): {line: number,
   return {line, column};
 }
 
-export interface CompilerOptions {
-  useTransformer: boolean;
-  /**
-   * Controls the name of the file produced by the compiler.  If there's more
-   * than one input file, they'll all be concatenated together in the outFile
-   */
-  outFile?: string;
-  filesNotToProcess: Set<string>;
-  inlineSourceMap: boolean;
-  generateDTS: boolean;
+export function assertSourceMapping(
```

Comment:
I like the current order.  It's `(stuffToFindTheOrginalPostion, expected)`, which matches the `assert(actual).equal(expected)` order.

And sure, sourceSnippet is better.